ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/node_type_keys.schema.yml

```diff
@@ -164,11 +164,20 @@ keys:
         uplink_type:
           type: str
           valid_values:
-            - "p2p"
-            - "port-channel"
-          default: "p2p"
+            - p2p
+            - port-channel
+            - p2p-vrfs
+          default: p2p
           description: |-
-            `uplink_type` must be "p2p" if `vtep` or `underlay_router` is true.
+            `uplink_type` must be `p2p` or `p2p-vrfs` if `vtep` or `underlay_router` is true.
+
+            For `p2p-vrfs`, the uplink are configured as an L3 interface with a subinterface for each VRF
```

Comment:
```suggestion
            For `p2p-vrfs`, the uplinks are configured as an L3 interface with a subinterface for each VRF
```

Reply:
Actually this should have been `uplinks are`

Reply:
I went back and forth on that, actually... initially, I had `uplinks are,` and I went back and changed  since the key is called  `uplink_type` "singular". But both work and I don't have any strong feelings on this....